content/using/raspberrypi/howtoguideraspberrypi.md

```diff
@@ -0,0 +1,187 @@
++++
+title = "How-To Guide for Raspberry Pi"
+description = "How to run Urbit on a Raspberry Pi."
+template = "doc.html"
+weight = 3
+[extra]
+hidetitle = "true"
++++
+
+## Introduction
+This how-to guide will walk you through setting up your own planet on a Raspberry Pi 4 in a cooling case with an optimally configured SATA SSD connected via the USB 3 port. Urbit will run great, and you’ll have enough integrated high-speed storage to run other Urbit-related applications on the same device in the future if you wish (e.g. a Bitcoin full node, Lightning node, etc).
+
+## Components
+Don’t attempt to run Urbit on anything less powerful than a Raspberry Pi 4. And never run Urbit on a microSD card or on a drive plugged into a USB 2 port. Performance will be suboptimal.
+
+The components and configuration in this how-to guide have been thoroughly tested and selected for ideal performance.
+
+**Component checklist:**
+
+- [Western Digital Blue 1TB M.2 SATA SSD](https://www.amazon.com/Blue-NAND-1TB-SSD-WDS100T2B0B/dp/B073SB2MXT/)
+- [Argon ONE M.2 Case for Raspberry Pi 4](https://www.argon40.com/argon-one-m-2-case-for-raspberry-pi-4.html)
+- [Argon ONE Pi 4 Power Supply](https://www.argon40.com/argon-one-power-supply-for-raspberry-pi-4.html)
+- [Raspberry Pi 4 8GB - board only (https://www.canakit.com/raspberry-pi-4-8gb.html)
+- [microSD card pre-installed with NOOBS - 64GB(https://www.canakit.com/raspberry-pi-sd-card-noobs.html)
+
+Total cost will be about $250.
+
+## Assembling the Hardware
+
+Follow the instructions included with the Argon case and assemble it, adding your Raspberry Pi 4 motherboard, your unformatted 1TB M.2 SATA SSD and your microSD card that has been loaded with [NOOBS](https://www.raspberrypi.org/downloads/%20NOOBS/).  
+
+Screenshots from the Argon manual are included below for your convenience.
+
+- https://i.gyazo.com/a9e5cc7969d46b2b122847541d6076c0.jpg
+- https://i.gyazo.com/e4fc630fa4f24957d22bace76be05554.jpg
+- https://i.gyazo.com/8882b82fdd51ffb08f290faef88a30ff.jpg
+- https://i.gyazo.com/09638586f05ceb98fbafe180f9052271.jpg
+- https://i.gyazo.com/4d4799a18a5ed26b988ade5c639d5e0b.jpg
+- https://i.gyazo.com/13fa0c9c95510df36664144458e06ab8.jpg
+- https://i.gyazo.com/5481a414e570404aed6c8ec6510cdfac.jpg
+- https://i.gyazo.com/97309a012a17787918730d8a7a5bf3f7.jpg
+- https://i.gyazo.com/e5e4b3635fbcb7a22a44aec9bac180b6.jpg
+- https://i.gyazo.com/05efc9bb01243a190e4e70bbb7f44498.jpg
+- https://i.gyazo.com/6058cd144da304c50a6603fd799cb01f.jpg
```

Comment:
It would be much better to include these as a separate document, probably called "Argon Manual", and then embedded directly into the site. 

We also generally prefer to host images in our own bucket. I'd suggest the following:

1. Rename each image to something that can be easily referenced (e.g. argon_manual_1.jpg, argon_manual_2.jpg, etc.).
2. Place all images into a single zip archive and send that to me (josh@urbit.org). I'll upload them all to our media bucket.
3. Change each link to be an embedded image pointing to `https://media.urbit.org/docs/raspberrypi/<image_url_here>`